src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketScmFormValidationDelegate.java

```diff
@@ -116,7 +116,7 @@ public FormValidation doCheckServerId(String serverId) {
         Jenkins.get().checkPermission(CONFIGURE);
         // Users can only demur in providing a server name if none are available to select
         if (bitbucketPluginConfiguration.getValidServerList().stream().noneMatch(server -> server.getId().equals(serverId))) {
-            return FormValidation.error("Please specify a valid Bitbucket Server.");
+            return FormValidation.error("Required");
```

Comment:
Same reasoning as before. Because the user sees the label, help text, and validation all at once I think it is better to simplify here and to leave this as is.